cpp/cmake/cudf-config.cmake.in

```diff
@@ -1,7 +1,68 @@
 @PACKAGE_INIT@
 
+
+#[=======================================================================[
+
+Provide targets for the cudf library.
+
+Built based on the Apache Arrow columnar memory format, cuDF is a GPU DataFrame
+library for loading, joining, aggregating, filtering, and otherwise
+manipulating data.
+
+cuDF provides a pandas-like API that will be familiar to data engineers &
+data scientists, so they can use it to easily accelerate their workflows
+without going into the details of CUDA programming.
+
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+If cudf is found, this module defines the following IMPORTED GLOBAL
+targets:
+
+ cudf::cudf             - The main cudf library.
+
+This module offers an optional testing component which defines the
+following IMPORTED GLOBAL  targets:
+
+ cudf::cudftestutil     - The main cudf testing library
+ cudf::gmock_main
+ cudf::gtest_main
+
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables in your project::
+
+  cudf_FOUND
+  CUDF_VERSION
+  CUDF_VERSION_MAJOR
+  CUDF_VERSION_MINOR
+
+#]=======================================================================]
+
+
 cmake_minimum_required(VERSION 3.18)
 
+set(_possible_targets_to_promote
+      cudf::cudf
+      cudf::benchmark
+      cudf::benchmark_main
+      cudf::gmock
+      cudf::gtest
+      cudf::gmock_main
+      cudf::gtest_main
```

Comment:
I believe we don't alias these but instead just publicly link to `GTest::gtest` for example

Reply:
We install them into the `cudf` namespace ( https://github.com/rapidsai/cudf/blob/d500142e91c041a69b531d06f346e1630a8af046/cpp/cmake/thirdparty/CUDF_GetGTest.cmake#L42 ). That causes our installed targets to link against our namespaced versions.